share/tools/web_config/themes/none.theme

```diff
@@ -0,0 +1,31 @@
+# name: '(Almost) No Colors'
+# preferred_background: 'black'
+
+fish_color_autosuggestion brblack
+fish_color_cancel
+fish_color_command
+fish_color_comment
+fish_color_cwd normal
+fish_color_cwd_root normal
+fish_color_end
+fish_color_error
+fish_color_escape
+fish_color_history_current
+fish_color_host normal
+fish_color_keyword
+fish_color_match
+fish_color_normal normal
+fish_color_operator
+fish_color_param
+fish_color_quote
+fish_color_redirection
+fish_color_search_match -r
+fish_color_selection -r
+fish_color_status normal
+fish_color_user normal
+fish_color_valid_path
+fish_pager_color_completion normal
+fish_pager_color_description brblack
+fish_pager_color_prefix '--underline'
+fish_pager_color_progress brblack
+fish_pager_color_selected_background --background=brblack
```

Comment:
I think this should be

```suggestion
fish_pager_color_selected_background -r
```

Otherwise I can't read the selected completion (+ description) in the pager

Reply:
I'm using gnome-terminal which likes to draw bright colors a little bit darker than other terminals :confused: 

![image](https://user-images.githubusercontent.com/6853872/139731238-1b4c4888-9f7a-4b83-87a3-99f6536a4848.png)

with `-r`

![image](https://user-images.githubusercontent.com/6853872/139731375-24ff61c6-e82d-4c13-a21d-564231d68fce.png)

hmm ok to get rid of the inconsistency with the description, we could use  `set fish_pager_color_description normal` on top of that. But that removes the nice gray tone for descriptions..